db/migrate/20230823115839_create_journals.rb

```diff
@@ -0,0 +1,22 @@
+class CreateJournals < ActiveRecord::Migration[7.0]
+
+	def up
+		create_table :journals, id: false do |t|
+			t.integer :journal_id, primary_key: true
+			t.integer :user_id 
```

Comment:
is this the foreign key ? If so, then please define it as follows
```suggestion
			t.integer :user_id, foreign_key: true
```